### ISSUE TITLE
[A11y] Mobile Explore Collections-List: My tab

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#2005]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

No

### Issue Description

Explore Collections-List: https://staging.oregondigital.org/explore_collections/all?locale=en

As a non logged in user, I see that My Collection tab is available on Mobile and Desktop. When I click on it I get the "Ack" message. Should the user be seeing this by default?

### Can anybody QA or does this need an expert?

Anybody

**Comment by azle** [comment]
@CGillen Original post for error is fixed. However, when validating found 2 errors, see below images

1 missing form label
<img width="1272" alt="Screen Shot 2021-12-13 at 8 46 13 AM" src="https://user-images.githubusercontent.com/8353860/145853448-f401fc30-9702-44e9-ad2c-104fae3ca6c8.png">

1 broken aria reference
<img width="975" alt="Screen Shot 2021-12-13 at 8 46 49 AM" src="https://user-images.githubusercontent.com/8353860/145853549-bed91d74-1f13-47d6-801d-01f27b1f7bf6.png">
